system/lvol.py

```diff
@@ -382,6 +395,22 @@ def main():
                     else:
                         module.fail_json(msg="Unable to resize %s to %s%s" % (lv, size, size_unit), rc=rc, err=err)
 
+    if this_lv is not None:
```

Comment:
Looks like if we want to activate/deactivate already present volume without specifying size,
like `lvol: vg=firefly lv=test active=no`, we won't get there because of check at line 289. Could you please tell me if I'm missing something, or this is intended?